scheduler/utilization_based_host_allocator.go

```diff
@@ -26,18 +26,18 @@ type TaskGroupData struct {
 
 func UtilizationBasedHostAllocator(ctx context.Context, hostAllocatorData HostAllocatorData) (int, error) {
 	distro := hostAllocatorData.Distro
-	if len(hostAllocatorData.ExistingHosts) >= distro.PoolSize {
+	nExistingHosts := len(hostAllocatorData.ExistingHosts)
+	if nExistingHosts >= distro.PoolSize {
 		return 0, nil
 	}
 
 	// split tasks/hosts by task group (including those with no group) and find # of hosts needed for each
-	nNewHosts := 0
 	taskGroupingBeginsAt := time.Now()
-
 	taskGroupDatas := groupByTaskGroup(hostAllocatorData.ExistingHosts, hostAllocatorData.DistroQueueInfo)
-	taskGroupingRuntime := time.Since(taskGroupingBeginsAt)
+	taskGroupingDuration := time.Since(taskGroupingBeginsAt)
 
-	calcNumHostsBeginsAt := time.Now()
+	nNewHostsRequired := 0
+	allocationCalcBeginsAt := time.Now()
```

Comment:
I think I added this before in case this operation was slow, but it's not needed now. I would remove this timing data (but keep the other log info)

Reply:
Removing.